packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -56,9 +57,10 @@ const trace = makeTracer('VM');
 /**
  * @typedef {object} MetricsNotification
  *
- * @property {number}         numVaults        present count of vaults
- * @property {Amount<'nat'>}  totalCollateral  present sum of collateral across all vaults
- * @property {Amount<'nat'>}  totalDebt        present sum of debt across all vaults
+ * @property {number}         numVaults          present count of vaults
+ * @property {number}         liquidatingVaults  present count of liquidating vaults
```

Comment:
these names are inconsistent. they should both or neither have `num` prefix. I request that it be included as a cue for "present count" but open to counter arguments.

Reply:
separately, the names and comment imply that one is a subset of the other but in code they're complements. so either rename to `numActiveVaults` or make `numVaults` include both the cardinality of both stores. I.e. the number of unsettled vaults.